lib/layout/force-directed-layout.js

```diff
@@ -0,0 +1,202 @@
+/**
```

Comment:
Please run this file through a javascript linter such as eslint. Settings for ESlint can be found in the style guide.